Intersect.Server/Maps/MapInstance.cs

```diff
@@ -771,12 +780,25 @@ public void Update(long timeMs)
                 //Process Items
                 lock (MapItems)
                 {
+
                     for (var i = 0; i < MapItems.Count; i++)
                     {
-                        if (MapItems[i] != null && MapItems[i].DespawnTime != -1 && MapItems[i].DespawnTime < timeMs)
+                        if (MapItems[i] != null)
```

Comment:
`var mapItem = MapItems[i]` and then reuse it in place of repeated `MapItems[i]`.